superset/databases/api.py

```diff
@@ -839,3 +841,67 @@ def function_names(self, pk: int) -> Response:
         if not database:
             return self.response_404()
         return self.response(200, function_names=database.function_names,)
+
+    @expose("/available/", methods=["GET"])
```

Comment:
My initial thought is that this endpoint refers to a status of databases and not a new resource. Should we make this endpoint more restful and if so, how about something like GET /databases?status=available. On the other hand, I realize that we already have a lot of other existing routes that aren't particularly restful either, so then this route would be an outlier.

Reply:
I don't have a strong opinion about this... @dpgaspar any thoughts here?

Reply:
looks like we can leave this as is for consistency.